src/charts/data-manager.js

```diff
@@ -245,5 +251,6 @@ export function getNearestDataPoint (_mouseX, _dataObject, _scales, _keyType) {
 
 function convertToDate (_date) {
   // hacks to handle invalid date like "0014-06-08T00:00:00.000Z"
-  return new Date(new Date(_date).toString())
+  // return new Date(new Date(_date).toString())
+  return new Date(_date)
 }
```

Comment:
Is this comment out of date? I'm unsure if the hack is persisting in the new code, and the commented out code should probably be dropped for clarity's sake.

Reply:
Changed back to match `master`.